man/systemd.exec.xml

```diff
@@ -1187,7 +1187,76 @@
         <function>read</function> and <function>write</function>, and
         right after it add a blacklisting of
         <function>write</function>, then <function>write</function>
-        will be removed from the set.) </para></listitem>
+        will be removed from the set.)</para>
+
+        <para>As the number of possible system
+        calls is large, predefined sets of system calls are provided.
+        A set starts with <literal>@</literal> character, followed by
+        name of the set.
+
+        <table>
+          <title>Currently predefined system call sets</title>
+
+          <tgroup cols='2'>
+            <colspec colname='set' />
+            <colspec colname='description' />
+            <thead>
+              <row>
+                <entry>Set</entry>
+                <entry>Description</entry>
+              </row>
+            </thead>
+            <tbody>
+              <row>
+                <entry>@clock</entry>
+                <entry>System clock setting</entry>
+              </row>
+              <row>
+                <entry>@default</entry>
+                <entry>Above mentioned default list</entry>
+              </row>
+              <row>
+                <entry>@event-loop</entry>
+                <entry>Event loop use (poll/select)</entry>
+              </row>
+              <row>
+                <entry>@ipc</entry>
+                <entry>Message queues, SYSV IPC or other IPC</entry>
+              </row>
+              <row>
+                <entry>@module</entry>
+                <entry>Kernel module control</entry>
+              </row>
+              <row>
+                <entry>@mount</entry>
+                <entry>Mounting</entry>
+              </row>
+              <row>
+                <entry>@network-io</entry>
+                <entry>Network or Unix socket IO</entry>
```

Comment:
I think we should clarify that "Unix" here means AF_UNIX. Maybe like this: "Socket I/O (including local AF_UNIX)".